sample-operators/mysql-schema/src/main/java/io/javaoperatorsdk/operator/sample/dependent/SchemaDependentResource.java

```diff
@@ -23,12 +28,17 @@
 import static io.javaoperatorsdk.operator.sample.dependent.SecretDependentResource.MYSQL_SECRET_USERNAME;
 import static java.lang.String.format;
 
+@SchemaConfig(pollPeriod = 700, host = "127.0.0.1",
+    port = SchemaDependentResource.LOCAL_PORT,
+    user = "root", password = "password") // NOSONAR: password is only used locally, example only
```

Comment:
We should have real life examples as much it makes sense, could we fix that here ? And show how to pass the password from the config override (and remove from the annotation) ?

Reply:
The whole point here is to show how the dependent resource can be configured from the annotation. I don't have time to create another example.

Reply:
I don't mean creating an another example, just passing the password from for the controller as it should be and not setting here by annotation.

Reply:
Again, the whole point is to show how the configuration can now be done via the annotation. I agree that passing the password in the annotation is dumb but that's what the configuration for this particular example is. Feel free to provide an alternative.